Engine/src/Engine/DeltaTime.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+namespace Engine
+{
+	//Container for the DeltaTime used in game.
+	class DeltaTime
+	{
+	public:
+		DeltaTime(double dt = 0)
+		: m_Time(dt)
+		{
+		}
+
```

Comment:
What does this do? Do I just get the current time or something?

Reply:
It's an overloaded operator for implicit conversions.

Reply:
can you provide context on how it's used? I'm not too familiar with this

Reply:
I'd be glad to explain how overloading types in cpp works for anyone that's interested, but I don't think comments in code should explain language features or programming concepts.

Reply:
That's fair, I'm just curious how this would be used if it's just returning a double again. But that's probably due to my lack of knowledge. I'll dismiss this for now if you really think no one would glean anything from a comment.